phpstan.neon

```diff
@@ -0,0 +1,5 @@
+parameters:
+	level: 1
+	tmpDir: /var/tmp
```

Comment:
The problem with this is that we all generate our own entries with the same filepaths, and by default can't overwrite each other's stuff. For purposes of development, we seem to need separated space since it doesn't guarantee unique file path naming.

Reply:
I think a better place for it is our own per-user pelagos/var directories.

Reply:
It can be a relative path, we could use var/tmp (in pelagos dir)